### DISCUSSION TITLE
Adding Skelton stops Flowbite-svelte Tooltip working

### DISCUSSION THREAD

**Discussion by VaughanG** [skeletonlabs/skeleton#619] [Ideas]
### Current Behavior

I can add Flowbite-Svelte to a Sveltekit app and use all the Flowbite-Svelte components without issue.
After I add Skeleton (following the guide) the Flowbite-Svelte Tooltip stops working.
I raised an issue with the Flowbite-Svelte team (https://github.com/themesberg/flowbite-svelte/issues/456) but they don't seem to have an answer. It would seem that something
skeletony has knobbled the Tooltip.
Hope someone can help.

### Steps To Reproduce

Create a new app with _npm create skeleton-app_.
Add Flowbite-Svelte with _npm i -D flowbite flowbite-svelte classnames @popperjs/core_
Update tailwind.config.cjs:
```
const config = {
	darkMode: 'class',
	content: [
		'./src/**/*.{html,js,svelte,ts}',
		'./node_modules/flowbite-svelte/**/*.{html,js,svelte,ts}',
		require('path').join(require.resolve('@brainandbones/skeleton'), '../**/*.{html,js,svelte,ts}'),
		'./src/**/*.{html,js,svelte,ts}'
	],
	theme: {
		extend: {}
	},
	plugins: [
		require('flowbite/plugin'),
		require('@tailwindcss/typography'),
		require('@tailwindcss/forms'),
		require('@brainandbones/skeleton/tailwind/theme.cjs')
	]
};
module.exports = config;
```
I have placed the flowbite/plugin in all postions relative to the other plugins with no difference.
+page.svelte
```
<script>
	import { Divider } from '@brainandbones/skeleton';
	
	import { Tooltip, Button } from 'flowbite-svelte';
	//import { tooltip } from '@brainandbones/skeleton';
</script>

<div>
<Divider borderWidth="border-t-8" borderStyle="border-solid" borderColor="border-primary-400"/>
</div>
<br>


<Button>Green tooltip</Button>
<Tooltip  color="green">Tooltip content</Tooltip>
<!-- <button use:tooltip={{ content: 'Skeleton' }}>Trigger</button> -->
```

### Anything else?

I commented-out the Skeleton tooltip which works fine before testing the Flowbite-svelte Tooltip which doesn't work.

**Comment by endigo9740** [comment]
@VaughanG Thanks for letting us know. As far as I'm aware you're venturing into untested territory. I'd love to see full integration between Skeleton and Flowbite Svelte in the future. However, currently the only advisable route to integrate Flowbite is via their standard HTML/CSS components (similar to what we call "Tailwind Elements") and introducing the Skeleton theme system:
https://flowbite.com/#components

We provide a mini-guide for this here:
https://www.skeleton.dev/docs/comparisons
 
Flowbite Svelte introduces component and logic and with that comes the potential for issues like you're seeing. Currently I would not advise using both libraries in this regard. If there's a particular component they provide you cannot live without let me know though. I can't promise an ETA but I will keep this in mind going forward.

**Comment by endigo9740** [comment]
FYI bug assumes "undesired behavior" where as this is more inline with a feature request, so I've updated the ticket labels accordingly.

**Comment by VaughanG** [comment]
Thanks endigo9740, I was barking up the wrong tree and sorry about the mislabelled ticket. I will happily stick
to Skeleton from now on!

**Comment by endigo9740** [comment]
No worries at all, it's good to know people are interested in this combination. I've chatted a bit with Zoltan (head of Flowbite) and he seems open to some sort of collab like this in the future. I'm also aiming to create a more in-depth guide for combining their HTML/CSS components with Skeleton in a future blog tutorial post. So, all good. It's been on our mind!

**Comment by endigo9740** [comment]
@VaughanG FYI I'm trying to prune and cleanup our issue queue a bit. I've gone ahead and bumped your feature request issue here to a "ideas" category Discussion. Like I mentioned above, we do have this as a goal, just ETA is TBD. I'd estimate it will be post-v1.0 though.
